storybook/config/preview.tsx

```diff
@@ -57,7 +58,4 @@ export const parameters = {
     viewports,
   },
   viewMode: 'docs',
-  html: {
-    root: 'div[lang="nl"]',
```

Comment:
Why did you remove this? We do not want to show the decorators in the HTML tab, right?